lte/gateway/deploy/roles/dev_common/tasks/main.yml

```diff
@@ -432,18 +448,6 @@
     while sudo fuser /var/lib/apt/lists/lock >/dev/null 2>&1 ; do
       sleep 1
     done
```

Comment:
We can also remove the `Wait for APT Lock` step here

Reply:
Removed.